URL extraction bug fix (#18107)
demisto/content: Packs/CommonScripts/ReleaseNotes/1_6_44.md

#### Scripts
##### FormatURL

> - Added a function that removes square brackets from the end of the URL if there are any.

Reviewer: ```suggestion
- The script now removes a trailing square bracket from the URL if there is any.
```
